frontmatter_parsers.ts

```diff
@@ -0,0 +1,39 @@
+import { titleCase } from "./deps/case.ts"
```

Comment:
Have we considered using parsers that already exist for TS formatter? i.e. 

https://github.com/microsoft/tsdoc/tree/main/tsdoc

We can roll our own one but it also comes at the cost of maintainance. writing tests, fiixng bugs, etc...

Reply:
If we use the TS compiler programmatic API, it slows things down substantially. To roll our own key-value parser isn't the worst.